requirements-manual.txt

```diff
@@ -1,4 +1,6 @@
 pydantic==1.8.2
 requests==2.25.1
+types_requests==2.27.25
 black==21.8b0
-sentry-sdk==1.5.1
\ No newline at end of file
+sentry-sdk==1.5.11
```

Comment:
sentry-sdk [released a typing fix](https://github.com/getsentry/sentry-python/issues/1415) for an issue run into [in CI](https://github.com/microCOVID/microCOVID/runs/6519153658?check_suite_focus=true)